browser/BUILD.gn

```diff
@@ -62,6 +63,9 @@ source_set("browser_process") {
     "component_updater/brave_component_updater_delegate.h",
     "geolocation/brave_geolocation_permission_context.cc",
     "geolocation/brave_geolocation_permission_context.h",
+    "infobars/brave_infobar_delegate.h",
```

Comment:
can we put these in a separate target and make it a dep of `browser_process`? We already have too much stuff in this one which is a problem because it's hard to track which deps belong to which code when things get removed/moved later

Reply:
The list is not that big at the moment... Especially compared to Chromium ones. IMO splitting things into many targets will not reduce the complexity at all, this would be just another sort of complexity (and copypasting deps between smaller targets)

Reply:
fixed